### PR TITLE
feat(front): footer base + globalstyle

### PR DESCRIPTION
**Pull request by OlivPi**

### CODE REVIEW

File: apps/frontend/src/assets/Logo/DatatlasLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
```
Comment (lutangar) [inline apps/frontend/src/assets/Logo/DatatlasLogo.tsx]: You could have only one import line.

File: apps/frontend/src/assets/Logo/FranceRelanceLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```
Comment (lutangar) [inline apps/frontend/src/assets/Logo/FranceRelanceLogo.tsx]: Same remark as above.

File: apps/frontend/src/assets/Logo/GitghubLogo.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { SVGProps } from 'react';
+
+export const GitghubLogo = (props: SVGProps<SVGSVGElement>) => (
```
Comment (lutangar) [inline apps/frontend/src/assets/Logo/GitghubLogo.tsx]: Typo on component name and file.

File: apps/frontend/src/assets/Logo/MetropoleLogo.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
```
Comment (lutangar) [inline apps/frontend/src/assets/Logo/MetropoleLogo.tsx]: Same remark as above.

File: apps/frontend/src/assets/Logo/GitghubLogo.tsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```
Comment (lutangar) [inline apps/frontend/src/assets/Logo/GitghubLogo.tsx]: Same.

**Review by lutangar — CHANGES_REQUESTED** [review]
The `assets` folder should only contains static and not components.
Since logos are now components they should be in the `components/logo` folder instead.

**Review by Pipazoul — APPROVED** [review]

**Pull request merged**